src/Common/OmnichannelEndpoints.ts

```diff
@@ -24,4 +24,5 @@ export default class OmnichannelEndpoints {
   public static readonly LiveChatGetReconnectableChatsPath = "livechatconnector/auth/reconnectablechats";
   public static readonly LiveChatReconnectAvailabilityPath = "livechatconnector/reconnect";
   public static readonly LiveChatValidateAuthChatMapRecordPath = "livechatconnector/auth/validateauthchatmaprecord";
+  public static readonly SendTypingIndicatorPath = "ccs-typingindicator/livechat/sendtypingindicator";
```

Comment:
Why this pattern? It should just be the same as we have for incoming messages on LiveChatMessagingConnector

Reply:
done. Update this api endpoint